src/LinSys/LinSysMerger.h

```diff
@@ -1,7 +1,7 @@
 // *****************************************************************************
 /*!
   \file      src/LinSys/LinSysMerger.h
-  \author    J. Bakosi
+  \author    Los Alamos National Laboratory
```

Comment:
As this project becomes more complex and has more developers, having a giant list of authors on every function becomes unwieldy. 

Also, the functionality you are talking about is provided by git (and can be used with the github interface) https://git-scm.com/docs/git-blame